src/ixdat/readers/msrh_sec.py

```diff
@@ -0,0 +1,240 @@
+                SpectroECMeasurement.
+        """
+
+        # us pandas to load the data from the csv files into dataframes:
+        sec_df = pd.read_csv(path_to_file)
+        # ^ Note the first row, containing potential, will become the keys. The first
+        #   column, containing an arbitrary counter, is included in the data.
+        ref_df = pd.read_csv(path_to_ref_spec_file, names=["wavelength", "counts"])
+        jv_df = pd.read_csv(path_to_V_J_file, names=["v", "j"])
+
+        # The spectra need (i) the first colum with the arbitrary counter to be
+        #    discarded and (ii) axes switched so that wavelength is the inner axis
+        #    (axis=1). The outer axis (axis=0) then spans potential or, eq., time:
+        spectra = sec_df.to_numpy()[:, 1:].swapaxes(0, 1)
+        # The potential comes from the keys of that data, discarding the first column:
+        v = np.array([float(key) for key in sec_df.keys()])[1:]
+        # We get time from this potential and the scan rate, with a helper function:
+        t = calc_t_using_scan_rate(v, dvdt=scan_rate * 1e-3)
+        # If they didn't provide a tstamp, we have to prompt for it.
+        tstamp = tstamp or prompt_for_tstamp(path_to_file)
+        # Ready to define the measurement's TimeSeries:
+        tseries = TimeSeries(
+            "time from scan rate", unit_name="s", data=t, tstamp=tstamp
+        )
+
+        # The wavelength comes from the reference spectrum file.
+        wl = ref_df["wavelength"].to_numpy()
+        excess_wl_points = len(wl) - spectra.shape[1]
+        # ^ This is how many points to discard to line up with sec data
+        #   (1 or 2 points in the example data).
+        wl = wl[excess_wl_points:]
+        ref_signal = ref_df["counts"].to_numpy()[excess_wl_points:]
+
+        # Now we're ready to define all the spectrum DataSeries:
+
+        # wavelength is independent variable --> simple DataSeries
+        wl_series = DataSeries("wavelength / [nm]", "nm", wl)
+        # The reference spectrum spans a space defined by wavelength:
+        reference = Field(
+            "reference",
+            "counts",
+            axes_series=[wl_series],
+            data=ref_signal,
+        )
+        # The spectra span a space defined by time and wavelength:
+        spectra = Field(
+            name="spectra",
+            unit_name="counts",
+            axes_series=[tseries, wl_series],
+            data=spectra,
+        )
+
+        # Now we process the current and potential:
+        v_0 = jv_df["v"].to_numpy()  # ... but we'll actually use v from the sec data
+        j_0 = jv_df["j"].to_numpy() * 1e3  # 1e3 converts [A] to [mA]
+        if v_0[0] > v_0[-1]:  # Need the potential in the EC file to be increasing:
+            v_0 = np.flip(v_0)
+            j_0 = np.flip(j_0)
+        # Since the "real" potential is in the sec data, we need to interpolate the
```

Comment:
In the current version of data form labview, I found the potential in SEC is always one point less than that in J-V, so it is fine, but very good to have this for other cases.